src/site/twitch.tv/modules/emote-menu/EmoteMenuContext.ts

```diff
@@ -22,3 +22,16 @@ export function useEmoteMenuContext(): EmoteMenuContext {
 
 	return data;
 }
+
+export type EmoteMenuSortPropertyKey = "Name" | "Listed" | "Animated" | "Timestamp";
+
+type EmoteMenuSort = {
+	[key in EmoteMenuSortPropertyKey]: string;
+};
+
+export const emoteMenuSortProperties: EmoteMenuSort = {
+	Name: "Name",
+	Listed: "Listed",
+	Animated: "Animated",
+	Timestamp: "Date added",
```

Comment:
Keys should be lowercase

```suggestion
	name: "Name",
	listed: "Listed",
	animated: "Animated",
	timestamp: "Date added",
```